shared/api/instance.go

```diff
@@ -193,6 +193,12 @@ type Instance struct {
 	// The type of instance (container or virtual-machine)
 	// Example: container
 	Type string `json:"type" yaml:"type"`
+
+	// Instance project name
+	// Example: foo
+	//
+	// API extension: instance_all_projects
+	Project string `json:"project" yaml:"project"`
```

Comment:
This should not be necessary as the expectation is that `?recursion=3` will effectively return a `map[string][]api.InstanceFull`, so the individual Instance/InstanceFull entries don't need to duplicate the project name.